src/platform/nrfconnect/wifi/WiFiManager.cpp

```diff
@@ -73,6 +73,32 @@ NetworkCommissioning::WiFiScanResponse ToScanResponse(const wifi_scan_result * r
     return response;
 }
 
+// Matter expectations towards Wi-Fi version codes are unaligned with
+// what wpa_supplicant provides. This function maps supplicant codes
+// to the ones defined in the Matter spec (11.14.3.2. WiFiVersion enum)
+uint8_t MapToMatterWiFiVersionCode(wifi_link_mode wifiVersion)
+{
+    if (wifiVersion < WIFI_1 || wifiVersion > WIFI_6E)
+    {
+        ChipLogError(DeviceLayer, "Unsupported Wi-Fi version detected");
+        return 0xff;
```

Comment:
Matter doesn't defined such a value. It's better to return some default than unspecified magic value.

Reply:
Yes, but what should be a default is such a case?